src/app/mytools/sidebar-accordion/sidebar-accordion.component.ts

```diff
@@ -44,8 +54,6 @@ export class SidebarAccordionComponent implements OnInit, OnChanges {
       if (this.registryToTools.has(groupEntryObject.registry)) {
         this.registryToTools.get(groupEntryObject.registry).groupEntryInfo.push(groupEntryObject);
       } else {
-        // If a tool doesn't belong to any of the specified keys then add to additional group
-        // For additional tools, remove once amazon and seven bridges have their own groups
```

Comment:
I'm getting a console error in my browser's developer tools for line 57: `ERROR TypeError: this.registryToTools.get(...) is undefined`. Probably because `additional` isn't in the map yet when this method is called so could add a check for that too